ql/src/java/org/apache/hadoop/hive/ql/ddl/table/storage/compact/AlterTableCompactAnalyzer.java

```diff
@@ -67,6 +73,18 @@ protected void analyzeCommand(TableName tableName, Map<String, String> partition
     }
 
     AlterTableCompactDesc desc = new AlterTableCompactDesc(tableName, partitionSpec, type, isBlocking, mapProp);
+//    Table table = getTable(tableName);
```

Comment:
nit: can we remove these comments as well?

Reply:
Sure